### PR TITLE
Fixes #2321 : I'm eating this should be in view mode, not edit mode

### LINKED ISSUE #2321: "I'm eating this" should be removed -> replaced by "Add to list"
image
I expected the button to be on the summary tab in view mode, not edit mode.

### PR DESCRIPTION
**Pull request by balaji-ramavathu**
## Description

Made "I'm eating this" checkbox in editing page from editing mode to view mode
Added a button in summary tab that toggles addition and removal of the product to eaten list.

## Related issues and discussion
#fixes #2321 
 
 ## Screen-shots, if any
 
eating1
eating2
eating3
eating4

### CODE REVIEW

File: app/src/main/res/layout/fragment_summary_product.xml [hunk 1]
```diff
@@ -136,6 +136,28 @@
...
 
+            <Button
+                android:id="@+id/btn_eating"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="8dp"
+                android:padding="8dp"
+                android:background="@drawable/button_grey_border"
+                android:drawableLeft="@drawable/ic_add_24dp"
+                android:text="I'm eating this" />
```
Comment (teolemon) [inline app/src/main/res/layout/fragment_summary_product.xml]: Can you extract the string to make it translatable ?

Comment (balaji-ramavathu) [reply app/src/main/res/layout/fragment_summary_product.xml]: Don't know why I forget this everytime :-P

**Comment by teolemon** [comment]
The blue one seems the best version of the 2. You can completely remove the checkbox from the edit mode.

**Comment by balaji-ramavathu** [comment]
I felt the same. Will remove it :-)

**Comment by teolemon** [comment]
```  
:app:compileObfDebugJavaWithJavac/home/travis/build/openfoodfacts/openfoodfacts-androidapp/app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AddProductOverviewFragment.java:314: error: cannot find symbol
            cbEating.setVisibility(View.GONE);
            ^
  symbol:   variable cbEating
  location: class AddProductOverviewFragment
```

**Comment by balaji-ramavathu** [comment]
I messed it up, so I'm going to close this and reopen another PR.

**Pull request closed without merging**